src/_nested-list.scss

```diff
@@ -0,0 +1,307 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+$block: #{$fd-namespace}-nested-list;
+
+@mixin fd-side-nav-arrow {
+  font-family: "SAP-icons";
+  font-style: normal;
+  font-weight: normal;
+  color: var(--sapList_TextColor);
+  font-size: var(--sapFontHeader4Size);
+  text-decoration: inherit;
+  text-transform: none;
+  speak: none;
+}
+
+.#{$block} {
+  @include fd-reset();
+
+  list-style: none;
+  width: 100%;
+
+  @for $i from 2 through 10 {
+    .level-#{$i} {
+      $paddingLeft: ($i - 2) + 2.75rem;
+
+      .#{$block}__link,
+      .#{$block}__content {
+        padding-left: $paddingLeft;
+        border-bottom: none;
+
+        @include fd-selected() {
+          border-bottom: var(--sapList_BorderWidth) solid;
+          border-bottom-color: var(--sapList_SelectionBorderColor);
+        }
+      }
+    }
+  }
+
+  @include fd-rtl() {
+    @for $i from 2 through 10 {
+      .level-#{$i} {
+        $paddingRight: ($i - 2) + 2.75rem;
+
+        .#{$block}__link,
+        .#{$block}__content {
+          padding-right: $paddingRight;
+          padding-left: 0;
+        }
+
+        .#{$block}__item {
+          border-bottom: none;
+
+          @include fd-selected() {
+            border-bottom: var(--sapList_BorderWidth) solid;
+            border-bottom-color: var(--sapList_SelectionBorderColor);
+          }
+        }
+      }
+    }
+  }
+
+  &[aria-hidden="true"] {
```

Comment:
I think we want an additional `.is-hidden` class to not fully rely on `aria` attributes for styles?

Reply:
so I think we never fully decided how we want to handle all the aria-selectors. We've got it implemented many ways across components with no real standard. We should figure this out in general, but maybe not here.